Mediapipe.Net/Mediapipe.Net.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup Label="Package References">
-    <PackageReference Include="Google.Protobuf" Version="3.19.3" />
+    <PackageReference Include="Google.Protobuf" Version="4.0.0-rc2" />
```

Comment:
Is that a prerelease version? I don't think we should use a version that's different from the one that our autogenerated protobufs use.

Reply:
Unless really needed, don't use pre-release versions, especially on things like Protobufs.

Reply:
Oh, my bad... i updated all the packages and it used a prerelease

Reply:
Bruh moment